updatecli/weekly.d/puppet-modules/firewall.yaml

```diff
@@ -0,0 +1,59 @@
+---
+name: Bump the Firewall Puppet Module
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  latestVersion:
+    kind: githubrelease
+    name: Get the latest puppetlabs-firewall module version
+    spec:
+      owner: puppetlabs
+      repository: puppetlabs-firewall
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionfilter:
+        kind: semver
+    transformers:
+      - trimprefix: v
+
+conditions:
+  testPuppetModuleExists:
+    kind: shell
+    disablesourceinput: true
+    spec:
+    # https://forge.puppet.com/v3/files/puppetlabs-firewall-4.0.0.tar.gz
```

Comment:
```suggestion
```
No value added? The curl command is explicit enough IMO.

Reply:
leftover that I forgot to remove 👍